<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileAttribute</class>
 <widget class="QDialog" name="fileAttribute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fileAttribute</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>408</width>
     <height>224</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0,0">
    <item row="0" column="2" colspan="3">
     <widget class="QLineEdit" name="lineEditFilename"/>
    </item>
    <item row="2" column="0" colspan="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>创建时间</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="btnFile">
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QCheckBox" name="checkBoxRead">
      <property name="text">
       <string>读</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QCheckBox" name="checkBoxLink">
      <property name="text">
       <string>软连接</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QCheckBox" name="checkBoxWrite">
      <property name="text">
       <string>写</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>最近修改时间</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="4">
     <widget class="QPushButton" name="btnGetFileAttr">
      <property name="text">
       <string>获取文件属性信息</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="checkBoxFile">
      <property name="text">
       <string>文件</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBoxDir">
      <property name="text">
       <string>目录</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="6">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>属性：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>文件名</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>大小</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QCheckBox" name="checkBoxExec">
      <property name="text">
       <string>执行</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="4">
     <widget class="QLabel" name="labelCreate">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>20</number>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="4">
     <widget class="QLabel" name="labelSize">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>20</number>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="4">
     <widget class="QLabel" name="labelModify">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>20</number>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
